setup.py

```diff
@@ -38,7 +38,7 @@
     ],
     packages=find_packages(include=["pulp_smash", "pulp_smash.*"]),
     install_requires=[
-        "click",
+        "click<8.0.0",
```

Comment:
The worry with this is that [pulpcore is using the newer version](https://github.com/pulp/pulpcore/blob/master/requirements.txt#L6) so this will cause it to receive a version incompatible when run there. I think the solution is to port pulp-smash to work with >8.0 and pin like pulpcore does.

Reply:
Here's how RQ fixed it FWIW https://github.com/rq/rq/pull/1465/files

Reply:
Yeah I actually thought this was the plan.  Which is my bad, I mis-read IRC.

Reply:
We're fixing this not only the wrong way, but in the wrong place - closing this PR :)

Reply:
Short-term, reopening to pin pulp-smash to 8.0.